src/assets/environment.json

```diff
@@ -2,7 +2,7 @@
   "environment": {
     "environment": "development",
     "apiRoot": "https://staging.ecosounds.org",
-    "siteRoot": "http://localhost:4200",
```

Comment:
Revert this change? Running this default environment file only works with `npm run start:dev`

Reply:
no this is a fine change - i've already suggested dev should bind to everything

maybe add a note to the readme that alerts the user that the dev server binds to all interfaces